rails_programming/advanced_forms_and_activerecord/active_record_callbacks.md

```diff
@@ -88,3 +88,13 @@ Callbacks are useful and many, like `:after_create` and `:before_destroy`, are p
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
 * [WikiBooks Reference on Callbacks](http://en.wikibooks.org/wiki/Ruby_on_Rails/ActiveRecord/Callbacks)
+
+### Knowledge Check
+This section contains questions for you to check your understanding of this lesson. If you're having trouble answering the questions below on your own, review the material above to find the answer.
+
+ * <a class='knowledge-check-link' href='https://guides.rubyonrails.org/active_record_callbacks.html#callback-registration'>How do you register a callback?</a>
+ * <a class='knowledge-check-link' href='https://guides.rubyonrails.org/active_record_callbacks.html#relational-callbacks'>How can you implement a user's images being destroyed if said user is destroyed?</a>
```

Comment:
This one would fit better in the association's lesson. In practice, the rails way ™️  of doing this would be by having a `dependent: :destroy` option on the user's images association instead of using a callback. Someone will probably bring that up if we keep this, they always do 😆

Can we remove this one please?